tests/e2e/progressive-pipeline-e2e/pipeline_test.go

```diff
@@ -289,28 +288,21 @@ var _ = Describe("Progressive Pipeline and ISBService E2E", Serial, func() {
 		VerifyPipelineRolloutInProgressStrategy(pipelineRolloutName, apiv1.UpgradeStrategyNoOp)
 		VerifyPipelineRolloutHealthy(pipelineRolloutName)
 
-		initialPipelineSpecVertices := []numaflowv1.AbstractVertex{}
-		for _, vertex := range initialPipelineSpec.Vertices {
-			initialPipelineSpecVertices = append(initialPipelineSpecVertices, numaflowv1.AbstractVertex{Name: vertex.Name, Scale: vertex.Scale})
-		}
-
-		initialPipelineSpecVerticesZero := []numaflowv1.AbstractVertex{}
-		for _, vertex := range initialPipelineSpec.Vertices {
-			initialPipelineSpecVerticesZero = append(initialPipelineSpecVerticesZero, numaflowv1.AbstractVertex{Name: vertex.Name, Scale: numaflowv1.Scale{Min: ptr.To(int32(0)), Max: ptr.To(int32(0))}})
-		}
-
-		By("Updating the Pipeline Topology to cause a Progressive change - Successful case")
+		By("Updating the Pipeline Topology to cause a Progressive change - Valid change")
```

Comment:
as a follow up PR, let's remove all Pipeline updates from this test. We should be able to simply update the isbsvc as bad -> the PipelineRollout and ISBServiceRollout should fail.